src/app/machines/views/MachineList/MachineList.tsx

```diff
@@ -87,6 +113,7 @@ const MachineList = ({
           {errorMessage}
         </Notification>
       ) : null}
+      {!headerFormOpen ? <ErrorsNotification errors={machinesErrors} /> : null}
```

Comment:
Do you think this component could also be used for the error message above (was that your intention with the `onAfterDismiss` prop)?